ground/gcs/src/plugins/setupwizard/setupwizard.cpp

```diff
@@ -366,3 +368,25 @@ bool SetupWizard::saveHardwareSettings() const
 
     return helper.setupHardwareSettings();
 }
+
+void SetupWizard::boardIgnorePrompt()
+{
+    if (!m_autoLaunched)
+        return;
+
+    QMessageBox prompt(QMessageBox::Question, tr("Ignore Board?"),
+                       tr("Would you like to ignore the setup wizard for this board?"),

```

Comment:
This should be something like.. "This board doesn't appear to be configured.  Would you like to automatically configure it now with the dRonin Setup Wizard?"